node/blockchain/serializers/signed_change_request.py

```diff
@@ -11,8 +11,17 @@ class SignedChangeRequestSerializer(serializers.Serializer, ValidateUnknownField
     signature = serializers.CharField()
     message = serializers.JSONField()
 
+    class Meta:
+        dataclass = SignedChangeRequest
+
     def create(self, validated_data):
         try:
-            return SignedChangeRequest.parse_obj(validated_data)
+            return self.Meta.dataclass.parse_obj(validated_data)
```

Comment:
`SignedChangeRequest.parse_obj(validated_data)` should behave as a factory method. Does not seem that the change is needed. Is it for API docs generation?

Reply:
You are right parse_obj method contained all required behavior.